.github/workflow_scripts/env_setup.sh

```diff
@@ -24,87 +24,44 @@ function setup_torch_gpu {
     python3 -m pip install torch==1.12.0+cu113 torchvision==0.13.0+cu113 --extra-index-url https://download.pytorch.org/whl/cu113
 }
 
-function install_common {
-    python3 -m pip install --upgrade -e common/[tests]
-}
-
-function install_core {
-    install_common
-    python3 -m pip install --upgrade -e core/
-}
-
-function install_core_all {
-    install_common
-    python3 -m pip install --upgrade -e core/[all]
-}
-
-function install_core_all_tests {
-    install_common
-    python3 -m pip install --upgrade -e core/[all,tests]
-}
-
-function install_features {
-    python3 -m pip install --upgrade -e features/
-}
-
-function install_eda {
-    python3 -m pip install --upgrade -e eda/[tests]
-}
-
-function install_fair {
-    python3 -m pip install --upgrade -e fair/
-}
-
-function install_tabular {
-    python3 -m pip install --upgrade -e tabular/[tests]
-}
-
-function install_tabular_all {
-    python3 -m pip install --upgrade -e tabular/[all,tests]
+function install_local_packages {
+    while(($#)) ; do
+        python3 -m pip install --upgrade -e $1
+        shift
+    done
 }
 
 function install_multimodal {
     # launch different process for each test to make sure memory is released
     python3 -m pip install --upgrade pytest-xdist
-    python3 -m pip install --upgrade -e multimodal/[tests]
+    install_local_packages "multimodal/$1"
```

Comment:
outside of the scope of this PR, but its a bit concerning we are doing special pip installs afterwards in our tests for multimodal... This isn't replicating the user experience. Especially using uncapped versions in the install.